run_wml_tests

```diff
@@ -168,6 +168,8 @@ class WesnothRunner:
                 path += "/wesnoth"
             if options.debug_bin:
                 path += "-debug"
+            elif not os.path.isfile(path) and sys.platform.startswith('linux'):
```

Comment:
If it's auto-detecting, then I feel `options.debug_bin` should become a tristate, with a default setting of "look for both binaries, error out instead of running tests if you find both".

Reply:
It's not so much auto-detecting as it's that cmake calls the output executables `wesnoth` for both Release and Debug builds, while scons adds the "-debug" suffix for debug builds.

Reply:
The two problems being:
* the CI doesn't have a good way as far as I know to use different arguments to the python processes depending on which is being used.
* locally, it's annoying since the shell scripts don't support running the -debug executables at all.

Reply:
Clarification - I'm happy with it checking which binary exists and automatically changing filename to the one that exists, just as long as it notices that both possible files exist, and errors out saying that it doesn't know which one to run.

Reply:
Fixed up a different way (just have scons not append that suffix).